src/utils/env.ts

```diff
@@ -1,15 +1,19 @@
-import { Client } from '@discordjs/core';
+import { Client, GatewayIntentBits } from '@discordjs/core';
 import { REST } from '@discordjs/rest';
 import { WebSocketManager } from '@discordjs/ws';
 import { env } from 'node:process';
 
 export const botToken = env.BOT_TOKEN ?? '';
 export const guildId = env.GUILD_ID ?? '';
-
+export const apiKey = env.IMAGE_API ?? '';
+export const apiUrl = env.IMAGE_URL ?? '';
```

Comment:
```suggestion
export const ocrSpaceKey = env.OCR_SPACE_API ?? '';
export const ocrSpaceUrl = env.OCR_SPACE_URL ?? '';
```
Let's improve the variable naming here. Will have to update some other files to reflect these changes.

Reply:
This is done.